src/panel/direction/DirectionPanel.jsx

```diff
@@ -114,22 +114,26 @@ export default class DirectionPanel extends React.Component {
         destination,
         vehicle,
       );
-      if (directionResponse && directionResponse.routes && directionResponse.routes.length > 0) {
-        const routes = directionResponse.routes.map((route, i) => ({
+      if (directionResponse
```

Comment:
It was supposed to be simpler, not more complex :P Why not just testing `directionResponse.error === 0` since you made all the job to uniformize it in the API?